minio/test.bats

```diff
@@ -0,0 +1,25 @@
+source ./plan.sh
+
+@test "Command is on path" {
+  [ "$(command -v gnatsd)" ]
```

Comment:
should this be minio, rather than gnatsd?

Reply:
Copy paste!